### ISSUE TITLE
Repair - Continuous loop after Drag Item to Inventory 

### ISSUE THREAD

**Issue by SleepingShadow-bit** [DinoDevs/GladiatusCrazyAddon#335]
Repair - Loop after Drag Item to Inventory.
U cant stop the loop - first and last way is to close the browser tab.

https://user-images.githubusercontent.com/8767665/148123044-384469f0-268c-4027-b2f1-40588a26ce9a.mp4

**Browser info (optional)**
 - OS: Windows,
 - Browser Firefox 
 - Version 95.0.2 (current Version)
 - Installed GCA version 4.3.5

Because of the error console.  I'm not sure where exactly I should open the console here.
No Errors ?

![Screenshot_1](https://user-images.githubusercontent.com/8767665/148123159-966dc284-7f18-453d-96de-828ef3377ae9.png)

**Comment by FrutyX** [comment]
This is not a GCA issue, it's Gladiatus server issue.

The servers tend to timeout when working with items, they just don't take it, I have this very often, with our without GCA.

Apo and Gram can take a look at it, but I know, that the game servers are slow like this already for some time.

It's this:
![20Lolwn](https://user-images.githubusercontent.com/23278552/148125753-638aefa4-40c5-4796-bee4-d149c27a47fd.png)

Similiar:
https://forum.gladiatus.gameforge.com/forum/thread/9500-smeltery-not-working-reload-loop/?postID=188554#post188554

**Comment by FrutyX** [comment]
I will not label this, nor close this.

Letting the real OG's to handle this, I know from my own experience that items can get into stuck loop, very often, but it could be a GCA bug, you never know.

Also if my mind does not betray me, Thanos was commenting a few months ago these server related problems on Gladiatus forum, if he remembers too..

**Comment by SleepingShadow-bit** [comment]
Let's wait and see what the guys say;)
Thanks for the link!

**Comment by FrutyX** [comment]
And are you getting this every single time? Test it out, keep your console open while doing it, in case you see an error.

**Comment by GramThanos** [comment]
> And are you getting this every single time? Test it out, keep your console open while doing it, in case you see an error.

@SleepingShadow-bit do this. If there is no GCA related error in the console, we can not know if this is a problem caused by the addon.

**Comment by FrutyX** [comment]
Veeeeery often.

`jquery.js:3 POST https://s34-cz.gladiatus.gameforge.com/game/ajax.php?mod=inventory&submod=move&from=512&fromX=3&fromY=1&to=5&toX=1&toY=1&amount=1&doll=5 net::ERR_CONNECTION_TIMED_OUT`

**Comment by FrutyX** [comment]
Oh my god, I went to properly test this on Firefox.

@GreatApo, @GramThanos this is insane.

When you move the item to your inventory, it goes completely insane, it reloads the page nonstop, stuck in a loop and you can't do anything, you can't click on anything, you can't do anything, your browser keeps refreshing the URL, only way out of this, is to close the Gladiatus tab completely.

https://user-images.githubusercontent.com/23278552/148952960-3bd1fa44-59ca-4292-8639-f69d425c3e6f.mp4

**Priority: CRITICAL**


@SleepingShadow-bit sorry for doubting you this time, I wanted to test it, but I was kinda busy these days, I also know that Gladiatus has a major server problem with items...

**Comment by SleepingShadow-bit** [comment]
I haven't had the time to test it extensively and intensively.  But I only know it with Firefox and only with an addon. 
Currently i am a little bit busy at work so there is no answer.  
But of course your video explains it better than mine.

**Comment by GramThanos** [comment]
It seems to me that no request is leaving the browser. The loop seems to try to create requests.

You can stop the loop by running inside the console the command `debugger`.
